internal/admission/handler.go

```diff
@@ -19,6 +19,8 @@ import (
 	kongv1beta1 "github.com/kong/kubernetes-ingress-controller/v3/pkg/apis/configuration/v1beta1"
 )
 
+const KindGateway = gatewayapi.Kind("Gateway")
```

Comment:
As far as I see, this consts is unused. Do we need it?

Reply:
Used in `httproute.go` L 121:
```
if parentRef.Kind != nil && *parentRef.Kind != "" && *parentRef.Kind != KindGateway
```